stdlib/src/utils/span.mojo

```diff
@@ -343,7 +347,7 @@ struct Span[
         for element in self:
             element[] = value
 
-    fn get_immutable(self) -> Span[T, _lit_mut_cast[origin, False].result]:
+    fn read(self) -> Span[T, _lit_mut_cast[origin, False].result]:
```

Comment:
**Question** Can you help me understand how the name `read()` is better/more favorable than the previous `get_immutable`?

Reply:
Just a matter of personal preference, I think it's shorter and self-explanatory. And it was really only used in 1 place so i figured nobody is using the API too much :man_shrugging: . Can revert it if you wish